podman/domain/system.py

```diff
@@ -51,11 +50,23 @@ def login(
             email: Registry account email address
             registry: URL for registry access. For example,
                 https://quay.io/v2
-            reauth: If True, refresh existing authentication. Default: False
-            dockercfg_path: Path to custom configuration file.
```

Comment:
This is the path `reauth` records the authentication information. it can also be ignored if not used.

Rather than delete the keyword, change the text to inform developers it is ignored.